### ISSUE TITLE
the training loss always between -0.01~1

### ISSUE THREAD

**Issue by timehasspeed** [MIC-DKFZ/nnUNet#101]
Dear Fabian,
  i have a trouble when training the model. The training loss is always between -0.01 to 1 and has been oscillating and not converging. I have check the loss function and can't find anything wrong.
Is that situation normal? Or how can i fix it ?
looking forward to your reply 
Best wishes

**Comment by timehasspeed** [comment]
this situation happens when you use 3d cascade unet ,it all because you make your batch size ==1 .dataloader goes different way when you use the dataset <2
